Tests/Source/LiftTester.cpp

```diff
@@ -0,0 +1,185 @@
+#include <string>
```

Comment:
Similarly these unit tests shouldn't have changed, perhaps exclude it from the commit.